### PR TITLE
chore: add volunteer kritikjain9

### PR DESCRIPTION
**Pull request by kritikjain9**

### CODE REVIEW

File: data/users/kritikjain9.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Kritik Jain",
+  "bio": "Frontend Dev",
+  "image": "https://twitter.com/_kritikjain/photo",
+  "categories": [
+    {
+      "slug": "programming | software-development | web-development",
+      "topics": []
```
Comment (codewithvoid) [inline data/users/kritikjain9.json:8]: no topics?

File: data/users/kritikjain9.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Kritik Jain",
+  "bio": "Frontend Dev",
+  "image": "https://twitter.com/_kritikjain/photo",
+  "categories": [
+    {
+      "slug": "programming | software-development | web-development",
```
Comment (codewithvoid) [inline data/users/kritikjain9.json:7]: you can only pick 1 category at a time

File: data/users/kritikjain9.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Kritik Jain",
+  "bio": "Frontend Dev",
+  "image": "https://twitter.com/_kritikjain/photo",
```
Comment (codewithvoid) [inline data/users/kritikjain9.json:4]: better put your github pfp here :)

**Comment by codewithvoid** [comment]
Thanks for volunteering @kritikjain9

File: data/users/kritikjain9.json (hunk 1 above)
Comment (kritikjain9) [reply data/users/kritikjain9.json:8]: I didn't completely understood the topics. What all can be the topics?

File: data/users/kritikjain9.json (hunk 2 above)
Comment (kritikjain9) [reply data/users/kritikjain9.json:7]: Sure, I'll make the changes

File: data/users/kritikjain9.json (hunk 3 above)
Comment (kritikjain9) [reply data/users/kritikjain9.json:4]: Yes, you are right. I checked the test deployment, and my profile picture wasn't showing up

**Comment by kritikjain9** [comment]
> Thanks for volunteering @kritikjain9

My pleasure :)

File: data/users/kritikjain9.json (hunk 1 above)
Comment (codewithvoid) [reply data/users/kritikjain9.json:8]: Theya are suppose to include specific skills. E.g for programming you can include python, javascript etc. For web development you can add NodeJS, React, Vue, etc

Comment (kritikjain9) [reply data/users/kritikjain9.json:8]: Oh, okay. Will add those and make a PR again

File: data/users/kritikjain9.json (hunk 3 above)
Comment (kritikjain9) [reply data/users/kritikjain9.json]: Hey @codewithvoid , I have updated the json file. Please let me know if you want me to make any other changes.

Thanks!

File: data/users/kritikjain9.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Kritik Jain",
+  "bio": "Frontend Dev",
+  "image": "https://avatars.githubusercontent.com/u/55712009?v=4",
+  "categories": [
+    {
+      "slug": "Software-development",
```
Comment (codewithvoid) [inline data/users/kritikjain9.json:7]: all lowercase `software-development`

File: data/users/kritikjain9.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+      "topics": [Javascript, React, NodeJS]
```
Comment (codewithvoid) [inline data/users/kritikjain9.json:8]: topics need to be in double quotes
```json
"topics": [JavaScript, React, NodeJS]
```

**Comment by codewithvoid** [comment]
hey @kritikjain9 added small comments. PTAL

**Comment by kritikjain9** [comment]
> hey @kritikjain9 added small comments. PTAL

Hey @codewithvoid, I have made the required changes. 

Thanks for your patience :)

**Comment by codewithvoid** [comment]
Perfect! Thanks for making the changes.

**Pull request merged**